docker/Dockerfile.minimal

```diff
@@ -5,7 +5,7 @@ ARG DOCKER_BRANCH=develop
 RUN mkdir /holochain
 
 WORKDIR /holochain
-ENV CARGO_HOME /holochain
+ENV CARGO_HOME /holochain/.cargo
```

Comment:
For some reason this doesn't work.  I did the build locally it's still building trycp_server into `/holochain/bin`.